xbmc/guilib/imagefactory.cpp

```diff
@@ -46,16 +46,15 @@ IImage* ImageFactory::CreateLoader(const CURL& url)
 
 IImage* ImageFactory::CreateLoaderFromMimeType(const std::string& strMimeType)
 {
-  VECADDONS codecs;
-  ADDON::CBinaryAddonCache &addonCache = CServiceBroker::GetBinaryAddonCache();
-  addonCache.GetAddons(codecs, ADDON::ADDON_IMAGEDECODER);
-  for (auto& codec : codecs)
```

Comment:
Does this mean that we will no longer need binary addon cache once all your changes are in? Is <code>CServiceBroker::GetBinaryAddonManager().GetAddonInfos()</code> performance-wise comparable with  <code>CServiceBroker::GetBinaryAddonCache().GetAddons()</code>?